azerothcore/azerothcore-wotlk
# (DB/Translations) Neutral quests in Grizzly Hills

## Changes Proposed:
-  Add the texts of the tables `quest_request_items_locale` and `quest_offer_reward_locale`. As well as some `ObjectiveText` from the `quest_template_locale` table that are untranslated.

## Issues Addressed:

- Closes

## List of translated quests

- [x] 11981 - Find Kurun!
- [x] 11982 - Raining Down Destruction
- [x] 11984 - Filling the Cages
- [x] 11985 - Into the Breach
- [x] 11989 - Truce?
- [x] 11990 - Vial of Visions
- [x] 11991 - Subject to Interpretation
- [x] 12007 - Sacrifices Must be Made
- [x] 12029 - Seared Scourge
- [x] 12038 - Seared Scourge
- [x] 12042 - Heart of the Ancients
- [x] 12068 - Voices From the Dust
- [x] 12070 - Rallying the Troops
- [x] 12081 - Gavrock
- [x] 12082 - Dun-da-Dun-tah!
- [x] 12093 - Runes of Compulsion
- [x] 12094 - Latent Power
- [x] 12099 - Free at Last
- [x] 12113 - Nice to Meat You
- [x] 12114 - Therapy
- [x] 12116 - It Takes Guts....
- [x] 12120 - Drak'aguul's Mallet
- [x] 12121 - See You on the Other Side
- [x] 12134 - Sasha's Hunt
- [x] 12137 - Chill Out, Mon
- [x] 12152 - Jin'arrak's End
- [x] 12190 - Say Hello to My Little Friend
- [x] 12279 - A Bear of an Appetite
- [x] 12327 - Out of Body Experience
- [x] 12328 - Ruuna's Request
- [x] 12329 - Fate and Coincidence
- [x] 12330 - Anatoly Will Talk
- [x] 12411 - A Sister's Pledge
- [x] 12483 - Shimmercap Stew
- [x] 12484 - Scourgekabob
- [x] 12802 - My Heart is in Your Hands

## SOURCE:


## Tests Performed:

- The tests were made inside the game, executing one by one the SQL queries, which I have them in separate files, but I put them together, all in one file, to make it easier to apply and test.
- At the time of executing the queries, both separately and as a whole, none of them presented conflicts.

## How to Test the Changes:


1. If you take missions from that area, you will notice, with the client in Spanish, many of the texts mentioned are untranslated.
2. You apply the SQL, delete the game client cache folder, and enter to test the quests.

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
